run

```diff
@@ -521,6 +529,12 @@ run_mkdrive() {
             local targetdir=$(realpath $5)
             local targetvol=(-v "$targetdir:$targetdir")
         fi
+        if [ "$(uname)" == "Darwin" ]; then
```

Comment:
It should be possible to use it for both platforms.

Reply:
done